draft-schwartz-svcb-dns.md

```diff
@@ -51,21 +51,19 @@ Names are formed using Port-Prefix Naming ({{SVCB}} Section 2.3).  For example,
 
 # Applicable existing SvcParamKeys
 
-## alpn and no-default-alpn
+## alpn
 
-These keys indicate the set of supported protocols ({{SVCB}} Section 6.1).  The default protocol is "dot", indicating support for DNS over TLS {{!DOT=RFC7858}}.
+This key indicates the set of supported protocols ({{SVCB}} Section 6.1).  There is no default protocol, so the `no-default-alpn` key does not apply.
 
 If the protocol set contains any HTTP versions (e.g. "h2", "h3"), then the record indicates support for DNS over HTTPS {{!DOH=RFC8484}}, and the "dohpath" key MUST be present ({{dohpath}}).  All keys specified for use with the HTTPS record are also permissible, and apply to the resulting HTTP connection.
 
 If the protocol set contains protocols with different default ports, and no port key is specified, then protocols are contacted separately on their default ports.  Note that in this configuration, ALPN negotiation does not defend against cross-protocol downgrade attacks.
 
-These keys are automatically mandatory if present.  (See Section 7 of {{SVCB}} for the definition of "automatically mandatory".)
```

Comment:
Should this be replaced by some text saying `alpn` is simply (not a technical term) mandatory now?

Reply:
Good point.  Done.